DigitalOffice/Services/ProjectService/Repositories/Interfaces/IGetProjectInfoByIdRepository.cs

```diff
@@ -0,0 +1,11 @@
+﻿using ProjectService.Database.Entities;
+using ProjectService.Models;
```

Comment:
Using directive is not required by the code and can be safely removed

Reply:
fixed